src/components/order-service/order-service.component.ts

```diff
@@ -2,7 +2,8 @@ import {OrderServiceController} from './order-service.controller';
 
 export const orderService: angular.IComponentOptions = {
   bindings: {
-    service: '<'
+    serviceClass: '<',
```

Comment:
'serviceClass'?  This could be a 'imageStream' as well, right?  I thought from the Catalog users can click on serviceClasses or imageStreams to order.